tests/e2e/tests/plugins/TypescriptPlugin.spec.ts

```diff
@@ -35,20 +34,19 @@ const topMenu: TopMenu = e2eContainer.get(CLASSES.TopMenu);
 const debugView: DebugView = e2eContainer.get(CLASSES.DebugView);
 const terminal: Terminal = e2eContainer.get(CLASSES.Terminal);
 const browserTabsUtil: BrowserTabsUtil = e2eContainer.get(CLASSES.BrowserTabsUtil);
-const workspaceNameHandler: WorkspaceNameHandler = e2eContainer.get(CLASSES.WorkspaceNameHandler);
 
-const devfileUrl: string = 'https://raw.githubusercontent.com/eclipse/che/main/tests/e2e/files/devfiles/plugins/TypescriptNodeDebug2PluginTest.yaml';
+const devfileUrl: string = 'https://github.com/SkorikSergey/web-nodejs-sample/tree/typescript-plugin';
 const factoryUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/f?url=${devfileUrl}`;
```

Comment:
Replace `devFileUrl` with `devfileUrl`

Reply:
it's `devfileUrl` already in the code, @makambalaji